src/lib/output/renderer.ts

```diff
@@ -201,7 +201,24 @@ export class Renderer extends ChildableComponent<Application, RendererComponent>
                 }
             }
 
-            this.theme = this.addComponent('theme', new DefaultTheme(this, path));
+            let filename = Path.join(path, 'theme.js');
+            if (!FS.existsSync(filename)) {
+                this.theme = this.addComponent('theme', new DefaultTheme(this, path));
+            } else {
+                try {
+                    const plugin = require(filename);
+                    if(typeof plugin === 'function') {
+                        plugin(this.application);
```

Comment:
This function is calling `plugin(this.application)` or `new plugin.default(this, path)`. These should be consistent, it shouldn't change depending on how someone exported it. To make it consistent with wherever else this is done, it'd probably be nice to put this in one place that does something like: `function req (id) { return typeof require(id) === 'function' ? require(id) : require(id).default }`. This way the behaviour is entirely consistent across the different export styles.